recipes/veloxchem/meta.yaml

```
@@ -0,0 +1,113 @@
+{% set name = "veloxchem" %}
+{% set version = "1.0" %}
+{% set pre = "rc3" %}
+{% set sha256 = "ed22aeb72d832f0daccffa393a1a349dab65fc85cefbb6a069933d3b51ad7ffd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/veloxchem/veloxchem/-/archive/v{{ version }}-{{ pre }}/veloxchem-v{{ version }}-{{ pre }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - patches/geometric-as-posix.patch
+
+build:
+  number: 0
+  script_env:
+    - OMPI_MCA_plm_rsh_agent=  # [mpi == 'openmpi' and linux]
+  binary_relocation: true
+  entry_points:
+    - vlx=veloxchem.main:main
+  ignore_run_exports:
+    - python
+  run_exports:
+    - {{ pin_subpackage('veloxchem', max_pin='x.x') }}
+
+requirements:
+  build:
+    - cmake >=3.18
+    - {{ compiler('cxx') }}
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx or win]
+    - ninja
+    - pkg-config   # [unix]
+    - gtest
+    - gmock
+    - pybind11 >=2.6,<3.0
+  host:
+    - python
+    - pybind11-abi
+    - {{ mpi }}
+    - mpi4py >=3.0
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
+    - numpy >=1.15
+    - xtb    # [unix]
+    - gtest  # [osx]
+    - gmock  # [osx]
+  run:
+    - psutil >=5.6
+    - mdanalysis >=1.0
+    - geometric
+    - cppe  # [unix]
+    - h5py >=2.8
+    - python
+    - {{ pin_compatible('mpi4py') }}
+    - {{ pin_compatible('numpy') }}
```

> By the way, the recipe is structured according to: https://conda-forge.org/docs/maintainer/knowledge_base.html#blas such that the BLAS/LAPACK implementation can be switched when installing. However, since the code uses OpenMP, only the OpenMP build of OpenBLAS is appropriate. Is there a way to enforce this in meta.yaml?

Yes, something like this:

```suggestion
    - {{ pin_compatible('numpy') }}
  run_constrained:
    - libopenblas * openmp*
```

---

Thanks!